rq/utils.py

```diff
@@ -26,6 +27,12 @@
 logger = logging.getLogger(__name__)
 
 
+class DequeueStrategy(str, Enum):
+    Order = "order"
```

Comment:
Let's follow Python's enum convention of using all caps for the options. `DequeueStrategy.ORDER`.

Having said that, just wondering if you have an alternative name for `DequeueStrategy.ORDER`?

Reply:
> Let's follow Python's enum convention of using all caps for the options. DequeueStrategy.ORDER.
Sure.

> wondering if you have an alternative name for DequeueStrategy.ORDER
I didn't like that name also tbh, but out of ideas on what to use. Though if leaving it as `None`, but it wouldn't be as obvious. Maybe `SEQUENCE` or something?

Reply:
@selwin suggested `DEFAULT` on the new version, let me know if you have any other suggestions.